### ISSUE TITLE
[docker image] There should be an image that contains plugin `image render`

### ISSUE THREAD

**Issue by jinleileiking** [grafana/grafana#25386]
**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by jinleileiking** [comment]
@peterholmberg  Is that not possible?

**Comment by jinleileiking** [comment]
Any ideas  ?
